src/components/Dashboard.js

```diff
@@ -18,19 +18,269 @@ import Tooltip from '@material-ui/core/Tooltip';
+          }}
+          component="div">
+          Welcome to HonkedIn!
+        </Typography>
+      </Box>
+
+      <Box sx={{ width: 1200, height: 1000, position: 'absolute', top: 250, left: '50%', transform: 'translateX(-50%)', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+        <Grid container spacing={10}>
+          <Grid item xs={8} md={4}>
+            <Typography
+              variant="h5"
+              align="center"
+              gutterBottom
+              component="div">
+              Recent Events
+            </Typography>
+            {events.map((event) => (
+              <Card style={{ width: '200 px', marginBottom: '20px' }} key={event.id}>
+                <CardContent>
+                  <Typography variant="h5" component="div">
+                    {event.name}<br />
+                  </Typography>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    Date of Event: {new Date(new Date(event.date).getTime() - (5 * 60 * 60 * 1000)).toLocaleDateString()}<br />
+                  </Typography>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    Location of Event: {event.location}<br />
+                  </Typography>
+                </CardContent>
+              </Card>
+            ))}
+            <Link to={`/events`}>
+              <Typography
+                variant="h6"
+                align="left"
+                gutterBottom
+                component="div">
+                View all events
+              </Typography>
+            </Link>
+          </Grid>
+          <Grid item xs={8} md={4}>
+            <Typography
+              variant="h5"
+              align="center"
+              gutterBottom
+              component="div">
+              Popular Forums
+            </Typography>
+            {forums.map((forum) => (
+              <Card style={{ width: '200 px', marginBottom: '20px' }} key={forum.id}>
+                <CardContent>
+                  <Link to={`/forum/${forum.id}`} target="_blank">
+                    <Typography variant="h5" component="div">
+                      {forum.forumTitle}<br />
+                    </Typography>
+                  </Link>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    {forum.description}<br />
+                  </Typography>
+                </CardContent>
+              </Card>
+            ))}
+            <Link to={`/forums`}>
+              <Typography
+                variant="h6"
+                align="left"
+                gutterBottom
+                component="div">
+                View all forums
+              </Typography>
+            </Link>
+          </Grid>
+          <Grid item xs={8} md={4}>
+            <Typography
+              variant="h5"
+              align="center"
+              gutterBottom
+              component="div">
+              Your RSVPed Events
```

Comment:
Maybe call it "Your Upcoming Events" to make it clear its only showing the ones that are coming up soon